src/Deanery.java

```diff
@@ -0,0 +1,111 @@
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+public class Deanery {
+    static DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
+    static LocalDateTime now = LocalDateTime.now();
+
+    private static String createRaport(Student student, String roomNumber) {
+        return "------------------------------------------" +
+                "\n" +
+                String.format("%20s", "STUDENT ") + student.getID().toString() +
+                "\n" + "\n" +
+                "First Name" + String.format("%10s", " ") + "|" + String.format("%3s", " ") + student.getName() + "\n" +
```

Comment:
Nice that you used a string format with %, but I guess a closing "|" is missing - I wanted it to look as a table so that you can work with the String format a bit. Imagine a Student with short first name and a super long second name. Then a table in the report would look a bit clumsy:
First Name      | Alan |
Second Name | VeryLongSecond Name |

and we are looking for something like that:
First Name      | Alan                                   | 
Second Name | VeryLongSecond Name    |